docs/content/guides/getting-started/react-redux.md

```diff
@@ -434,19 +434,17 @@ export const ExampleComponent = () => {
         autoWrapCol={true}
         licenseKey="non-commercial-and-evaluation"
       >
-        <HotColumn width={100} type={'numeric'}>
-          {/* add the `hot-renderer` attribute to mark the component as a Handsontable renderer */}
-          <StarRatingRenderer hot-renderer />
-        </HotColumn>
-        <HotColumn width={150}>
-          {/* add the `hot-renderer` attribute to mark the component as a Handsontable renderer */}
+        {/* add the `renderer` prop to set the component as a Handsontable renderer */}
+        <HotColumn width={100} type={'numeric'} renderer={StarRatingRenderer} />
+        {/* add the `renderer` prop to set the component as a Handsontable renderer */}
+        <HotColumn width={150} renderer={ColorPicker}>
           {/* add the `hot-editor` attribute to mark the component as a Handsontable editor */}
-          <ColorPicker hot-renderer hot-editor />
```

Comment:
This was an interesting use case, using the same component as both renderer and editor... It wouldn't work in the new approach. I'm not sure if that was something we should fight for, although if this opens a way to share the state between the render and edit stages, maybe it's something we actually overlooked? 🤔

Reply:
I was discussing this with a developer recently when I found this 😄, we are fine to ignore this case as its not known to be used by anyone.